sslsniff: cannot call non-static helper function min

sslsniff is failing to run complaining about the `min` function.

Using:
```
bcc 0.33.0
Linux arco 6.13.7-zen1-1-zen #1 ZEN SMP PREEMPT_DYNAMIC Thu, 13 Mar 2025 18:11:42 +0000 x86_64 GNU/Linux
```

```
❯ sudo /usr/share/bcc/tools/sslsniff -c curl
/virtual/main.c:82:29: error: cannot call non-static helper function
   82 |         u32 buf_copy_size = min((size_t)MAX_BUF_SIZE, (size_t)len);
      |                             ^
include/linux/minmax.h:129:19: note: expanded from macro 'min'
  129 | #define min(x, y)       __careful_cmp(min, x, y)
      |                         ^
include/linux/minmax.h:105:2: note: expanded from macro '__careful_cmp'
  105 |         __careful_cmp_once(op, x, y, __UNIQUE_ID(x_), __UNIQUE_ID(y_))
      |         ^
include/linux/minmax.h:100:2: note: expanded from macro '__careful_cmp_once'
  100 |         BUILD_BUG_ON_MSG(!__types_ok(x,y,ux,uy),        \
      |         ^
note: (skipping 2 expansions in backtrace; use -fmacro-backtrace-limit=0 to see all)
include/linux/compiler_types.h:530:2: note: expanded from macro '_compiletime_assert'
  530 |         __compiletime_assert(condition, msg, prefix, suffix)
      |         ^
include/linux/compiler_types.h:523:4: note: expanded from macro '__compiletime_assert'
  523 |                         prefix ## suffix();                             \
      |                         ^
<scratch space>:86:1: note: expanded from here
   86 | __compiletime_assert_554
      | ^
1 error generated.
Traceback (most recent call last):
  File "/usr/share/bcc/tools/sslsniff.orig", line 274, in <module>
    b = BPF(text=prog)
  File "/usr/lib/python3.13/site-packages/bcc/__init__.py", line 505, in __init__
    raise Exception("Failed to compile BPF module %s" % (src_file or "<text>"))
Exception: Failed to compile BPF module <text>
```

Incrementing debug level to 4 `b = BPF(text=prog, debug=4)`: 
[sslsniff_debug4.txt](https://github.com/user-attachments/files/19360582/sslsniff_debug4.txt)

--- a/tools/sslsniff.py
+++ b/tools/sslsniff.py
@@ -117,6 +117,11 @@ BPF_PERF_OUTPUT(perf_SSL_rw);
 BPF_HASH(start_ns, u32);
 BPF_HASH(bufs, u32, u64);
 
+__attribute__((always_inline))
+static inline u32 bpf_min(u32 a, u32 b) {
+  return (a < b) ? a : b;
+}
+
 int probe_SSL_rw_enter(struct pt_regs *ctx, void *ssl, void *buf, int num) {
         int ret;
         u32 zero = 0;
@@ -170,7 +175,7 @@ static int SSL_exit(struct pt_regs *ctx, int rw) {
         data->len = (u32)len;
         data->buf_filled = 0;
         data->rw = rw;
-        u32 buf_copy_size = min((size_t)MAX_BUF_SIZE, (size_t)len);
+        u32 buf_copy_size = bpf_min(MAX_BUF_SIZE, len);
 
         bpf_get_current_comm(&data->comm, sizeof(data->comm));
 
@@ -205,7 +210,7 @@ int probe_SSL_do_handshake_enter(struct pt_regs *ctx, void *ssl) {
         u32 tid = (u32)pid_tgid;
         u64 ts = bpf_ktime_get_ns();
         u32 uid = bpf_get_current_uid_gid();
-        
+
         PID_FILTER
         UID_FILTER
 

